### DISCUSSION TITLE
Using suspense and enabled together

### DISCUSSION THREAD

**Discussion by ValentinBond** [TanStack/query#6408] [Q&A]
Before version 5 I used the normal useQuery and specified the saspence flag to display the loader, sometimes I needed to specify the enable flag to run a chain of queries one by one. But in the new version we don't have the option to specify two flags together suspense and enabled. How can we handle cases when we need to load content not at once or handle a chain of requests while controlling the loader with suspense?

**Comment by TkDodo** [comment]
do you know that suspense works in a way that each call to `useSuspenseQuery` suspends the component from further rendering, so two `useSuspenseQuery` calls will run in serial per default?
